stable/database/files/nuosm

```diff
@@ -422,11 +409,11 @@ else
   log "Directory $DB_DIR exists"
 fi
 
-# ensure the admin layer is intact...
-checkAdminLayer 30
-
 trace "retrieving restore request from raft"
-legacy_restore_requested="$( nuocmd get value --key "$restore_req" )"
+legacy_restore_requested="$( nuocmd get value --key "$restore_req" 2>&1 )"
+if [ $? -ne 0 ]; then
+  die 1 "Unable to retrieve legacy restore request: ${legacy_restore_requested}"
+fi
```

Comment:
Do we have any convenience function in here to retry a command? If we really want to have the equivalent behavior to the previous health check, we could retry the `nuocmd get value` command a few times until 30 seconds have elapsed (I think there is an 8-second timeout, so there would be about 3 re-tries). But I'm not sure if we actually ever needed the original health check in the first place, so this might be fine.